url.bs

```diff
@@ -2378,11 +2377,11 @@ takes a byte sequence <var>input</var>, and then runs these steps:
 
 <ol>
  <li><p>Let <var>sequences</var> be the result of splitting <var>input</var> on
- `<code>&amp;</code>`.
- <!-- XXX define splitting? DOM does not do it -->
+ 0x26 (&amp;).
+ <!-- XXX define strictly splitting for byte sequences in Infra -->
```

Comment:
Could you add "or decode earlier" to this comment, since we haven't decided on which is better.